lib/recursorUtils.js

```diff
@@ -24,6 +24,22 @@ function fork (recursor) {
 }
 exports.fork = fork
 
+function greedy (recursor) {
```

Comment:
I'm definitely nitpicking/bikeshedding here, but I don't think `greedy` is the right name for this. `greedy(recursor)` sounds like it ought to return a recursor that's greedy, when it actually _greedily consumes_ the recursor and returns nothing. Using a verb name would better convey that to a reader.

Reply:
`consumeDeep`?